README.md

```diff
@@ -229,7 +229,14 @@ After the setup is complete, you'll receive an email with your JFrog environment
 </details>
 
 <details>
-  <summary>Step 2 - Install Frogbot</summary>
+  <summary>Step 2 - Create the frogbot-config.yml file if needed</summary>
```

Comment:
Maybe we should add a short sentence on which cases we'll need the config file?

Reply:
Since the reasons for when the frogbot-config.yml is mandatory requires more than one sentence, I'm not sure how we can squeeze this into one sense here.
I added it here, and I'll expand it more now.
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/7830056/224537197-61dec8b5-03d8-45a7-ab0c-11ce48179885.png">